### DISCUSSION TITLE
Kong 3.8.1 RPM package not available via cloudsmith.io

### DISCUSSION THREAD

**Discussion by heckea** [Kong/kong#14008] [Help]
Hi,
I have seen there was a tag for 3.8.1 some time ago that includes a fix for the rate-limiting plugin that we are waiting for. We normally use the RPM package at cloudsmith.io to build our own image including this artifact, but still there is not kong 3.8.1 available (but I have seen the enterprise version is there already).
Is there some issue, should we use another repo or is there something else to consider?
Many thanks in advance

**Comment by Water-Melon** [comment]
Hi @heckea ,
Which RPM package did you download? Could you provide a link?

I just tried the `3.8.1.0-1`(`el9`, `aarch64`) RPM package from https://cloudsmith.io/~kong/repos/gateway-38/packages/, and it worked without any issues in my dev environment.

**Reply by heckea** [reply]
Hi @Water-Melon , I guess you are talking about kong-enterprise-edition I see that this one is there. I was looking for plain kong, amazonlinux x86_64. Meanwhile, we saw that 3.9.0 is available and we are going to use that one, nevertheless it was somehow strange that 3.8.1 enterprise was there for quite some time and plain kong never appeared. Also the release itself in github looks different like the other ones, so I was thinking that there went something wrong in the release process.

**Reply by mschonmeier** [reply]
Same thing here @heckea. We need to use the version 3.8.1 community.

**Reply by heckea** [reply]
As stated back then, meanwhile we did the switch to 3.9.0, it fixes the issue and works quite well.
